src/apps/relay/mainrelay.c

```diff
@@ -3831,4 +3838,13 @@ static void reload_ssl_certs(evutil_socket_t sock, short events, void *args) {
   UNUSED_ARG(args);
 }
 
+static void shutdown_handler(evutil_socket_t sock, short events, void *args) {
+  TURN_LOG_FUNC(TURN_LOG_LEVEL_INFO, "Caught signal, terminating\n");
```

Comment:
We cannot know what signal was sent
Should we have different handlers (per signal) instead?
Will look like code duplication but eventually will give us opportunity to handle different signals in different way (not to mention proper logging)